test/server/hot_restarting_parent_test.cc

```diff
@@ -64,6 +65,48 @@ TEST_F(HotRestartingParentTest, GetListenSocketsForChildNotBindPort) {
   EXPECT_EQ(-1, message.reply().pass_listen_socket().fd());
 }
 
+TEST_F(HotRestartingParentTest, GetListenSocketsForChildSocketType) {
```

Comment:
It would be great to have a test for UNIX socket since I didn't find any existing test cover the UNIX socket also.

Reply:
I added a test for UNIX domain socket.